src/GitVersionCore/Configuration/ConfigurationBuilder.cs

```diff
@@ -0,0 +1,276 @@
+using System;
+using System.Collections.Generic;
+using GitVersion.Extensions;
+using GitVersion.Model.Configuration;
+using GitVersion.VersionCalculation;
+using JetBrains.Annotations;
+
+namespace GitVersion.Configuration
+{
+    public class ConfigurationBuilder
```

Comment:
Nice. Whenever you're ready for a new review, click the ↺ button.

<img width="241" alt="image" src="https://user-images.githubusercontent.com/12283/84001025-1c9c8700-a966-11ea-8125-4c50e2a7d857.png">